src/models/service/transactions/summary.rs

```diff
@@ -16,21 +16,27 @@ use serde::Serialize;
 ///       "type": "DATE_LABEL",
 ///       "timestamp": 1604620800000
 ///     },
-///     {
+///      {
 ///       "type": "TRANSACTION",
 ///       "transaction": {
 ///         "id": "multisig_0x126ab4d9e87b5cba98Ddeb75Df703E83500b6B7f_0x28b4cc29c036c2df40a1ba8d684cdab736abaf7d5cb84b217428462a2b4e3318",
 ///         "timestamp": 1604700419000,
 ///         "txStatus": "SUCCESS",
 ///         "txInfo": {
 ///           "type": "Custom",
-///           "to": "0x8D29bE29923b68abfDD21e541b9374737B49cdAD",
+///           "to": {
+///             "value": "0x8D29bE29923b68abfDD21e541b9374737B49cdAD",
+///             "name": "Gnosis Safe: Multi Send 1.1.1",
+///             "logoUri": "https://safe-transaction-assets.staging.gnosisdev.com/contracts/logos/0x8D29bE29923b68abfDD21e541b9374737B49cdAD.png"
```

Comment:
Do we have a production URI for this one?

Reply:
I am not sure, as I just pasted the json I took from the service itself. I think the value being from either staging or prod doesn't matter in the scope of documentation.